src/ui/Templates/CoreHelpers.ts

```diff
@@ -297,15 +297,34 @@ export interface ICoreHelpers {
 
 /**
  * Available options for the size templateHelpers.
+ * Example
```

Comment:
I think I'd rather add the example here:

https://coveo.github.io/search-ui/interfaces/icorehelpers.html#size